### ISSUE TITLE
Improve color contrast of syntax highlighting in code blocks

### ISSUE THREAD

**Issue by sivakumar-kailasam** [ember-learn/guides-app#97]
Our code blocks fail on color contrast tests. Especially the code blocks with diff in them.

**Comment by localpcguy** [comment]
I'll take a look at adjusting these colors.

**Comment by localpcguy** [comment]
First pass, looking for feedback:

HTML/HBS
![image](https://user-images.githubusercontent.com/593240/40447781-b0e69520-5ea1-11e8-8c0b-f702a4e807ec.png)

JavaScript
![image](https://user-images.githubusercontent.com/593240/40447823-ceba240e-5ea1-11e8-926d-03fc6bcbcbcb.png)

**Comment by sivakumar-kailasam** [comment]
LGTM
